examples/example_plugin/pyproject.toml

```diff
@@ -16,3 +16,14 @@ python = ">=3.8,<3.12"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.ruff]
+line-length = 120
+target-version = "py38"
+
+[tool.ruff.lint.isort]
+known-first-party = ["example_plugin"]
+section-order = ["future", "standard-library", "nautobot", "first-party", "local-folder", "third-party"]
```

Comment:
any reason the example is different than nautobot core?

Reply:
The `section-order` and `sections` below were to force imports from `nautobot.*` to be their own section in the imports listing. Probably would be good to add the other settings from core here as well though. Thanks!

Reply:
This is actually wrong - will fix :)